doc/api/buffer.md

```diff
@@ -404,14 +404,14 @@ to initialize a `Buffer` to zeroes.
 Example:
 
 ```js
-const buf = new Buffer(5);
+const buf = new Buffer(10);
```

Comment:
Sorry for my incomprehensibility. Consider this case: a user copies this example, pastes it in the empty file `test.js`, saves it and run via console or IDE to see the output. He will not see described output, he will see two identical lines with null bytes all the times. In #9876 I've described the cause. This fix increases the size of possible `test.js` so the two lines will be different. I'll try to make some GIFs now to explain more clearly what I mean.